experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -610,20 +610,24 @@ if ClimaComms.iamroot(comms_ctx)
             T = (:regrid, :zonal_mean),
             u = (:regrid, :zonal_mean),
             q_tot = (:regrid, :zonal_mean),
-            toa = (:regrid, :horizontal_slice),
+            toa_fluxes = (:regrid, :horizontal_slice),
             precipitation = (:regrid, :horizontal_slice),
             T_sfc = (:regrid, :horizontal_slice),
+            q_liq_ice = (:regrid, :zonal_mean),
+            tubulent_energy_fluxes = (:regrid, :horizontal_slice),
         )
 
         plot_spec = (;
             T = (; clims = (190, 320), units = "K"),
             u = (; clims = (-50, 50), units = "m/s"),
-            q_tot = (; clims = (0, 50), units = "g/kg"),
-            toa = (; clims = (-250, 210), units = "W/m^2"),
-            precipitation = (clims = (0, 1e-6), units = "kg/m^2/s"),
+            q_tot = (; clims = (0, 30), units = "g/kg"),
+            toa_fluxes = (; clims = (-250, 210), units = "W/m^2"),
+            precipitation = (clims = (0, 1e-4), units = "kg/m^2/s"),
             T_sfc = (clims = (225, 310), units = "K"),
+            q_liq_ice = (; clims = (0, 10), units = "g/kg"),
+            tubulent_energy_fluxes = (; clims = (-250, 210), units = "W/m^2"),
```

Comment:
Same as above comment on `clims` - would (-250,250) be useful here for visualizations?